src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplateUtils.java

```diff
@@ -201,6 +202,11 @@ public static Pod combine(Pod parent, Pod template) {
         List<Volume> combinedVolumes = Lists.newLinkedList();
         Optional.ofNullable(parent.getSpec().getVolumes()).ifPresent(combinedVolumes::addAll);
         Optional.ofNullable(template.getSpec().getVolumes()).ifPresent(combinedVolumes::addAll);
+        
+        // Tolerations
+        List<Toleration> combinedTolerations = Lists.newLinkedList();
+        Optional.ofNullable(parent.getSpec().getTolerations()).ifPresent(combinedTolerations::addAll);
+        Optional.ofNullable(template.getSpec().getTolerations()).ifPresent(combinedTolerations::addAll);
```

Comment:
This would never replace parent tolerations, always append. Is that the expected behavior ?
IIUC you can have multiple tolerations with the same key, but just checking

Reply:
Yeah, per https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/ you can have the same key with different contents ...several examples on that page that do that.

On whether to override or add ... for our uses cases at least, we can make some simplifying assumptions wrt the yaml field being the starting point, and most likely there won't be existing tolerations that we have to worry about overriding.

So I'm good with keeping it simple and not adding more levers around tuning the behaviour.  But if you want those let me know.

Reply:
Though upon re-reading the help text, it could be misleading wrt that ... I'll update it to clarify.